### PR TITLE
docs(rxModalAction): FRMW-184-Migrate rxModalAction to API Docs

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-184

- [x] Dev LGTM
- [x] Dev LGTM

Migrate `rxModalAction` to API Documentation.

### CODE REVIEW

**Comment by bry4n** [comment]
LGTM

File: src/rxModalAction/rxModalAction.js [hunk 1]
```diff
@@ -163,6 +177,20 @@ angular.module('encore.ui.rxModalAction', ['ui.bootstrap'])
...
  *
+ * When a modal has multiple views or kicks off a process that should be tracked within the modal, the 
+ * `<rx-modal-footer>` directive should be used.  
+ *
+ * Modal Footers should be defined in the same template as the Modal Form unless the footer is global, in which case it
+ * should be loaded in `module.run()`.  Global footers can be used in any subsequent modal by changing to the state 
+ * they were defined with.
+ *
+ * The modal's controller also inherits the `setState()` method on the scope, which should be used to toggle different 
+ * views or footers. See the *Multi-View Example* in the component {@link /#/components/rxModalAction Demo} of this
```
Comment (CITguy) [inline src/rxModalAction/rxModalAction.js]: * downcase "Demo"

```
{@link /#/components/rxModalAction demo}
```

**Comment by 100stacks** [comment]
@CITguy okay made the changes :+1:

**Comment by CITguy** [comment]
LGTM

@100stacks, please squash PR

**Comment by 100stacks** [comment]
@CITguy okay just completed. :+1:

**Pull request merged**